### ISSUE TITLE
gpt-3.5-turbo-0613 is being deprecated on 6/13/2024; need to update to gpt-3.5-turbo-instruct

### ISSUE THREAD

**Issue by princetrunks** [supabase-community/chatgpt-your-files#32]
I saw that if using the OpenAI apis that the model used in the chat function, gpt-3.5-turbo-0613 , will be soon deprecated and might not work.

As per this page:
https://platform.openai.com/docs/models/gpt-3-5-turbo

I confirmed on my instance that changing line 122 in supabase/functions/chat/index.ts to gpt-3.5-turbo-instruct works with uploaded documents/ previously saved embeddings.  Going to any other will not work and the chat will not know of the documents as it seems that the structure of completionStream or completionMessages might need to be refactored for newer models.  This from how OpenAI states that the gpt-3.5-turbo-instruct model still uses legacy completions.

**Comment by gregnr** [comment]
Good catch @princetrunks about the model deprecation - we should update this. I'm a bit surprised that `gpt-3.5-turbo-instruct` worked for you though since that uses the legacy completions endpoint vs. chat-based endpoints (which we [use in this repo](https://github.com/supabase-community/chatgpt-your-files/blob/0bc06b7b9522f2b22581f4fdf49218f05f702371/supabase/functions/chat/index.ts#L118-L124)). Did you have to modify any other code as well?

In other projects I've simply updated the model to the latest version. Any reason `gpt-3.5-turbo-0125` shouldn't work here from your perspective?

**Comment by princetrunks** [comment]
Hey @gregnr.  No problem.  Yeah, I was shocked it worked as I tried gpt-3.5-turbo-0125 prior and that oddly didn't work.  My completion object is the same.  I didn't log / save what the error was before I got instruct to work but I'll see if I can see what that might be.  My completion object is the same as the one in the repo.  I duplicated the functions and frontend so I'll do a comparison in my next push to my dev site.
